IPython/core/display.py

```diff
@@ -952,6 +972,10 @@ def _repr_jpeg_(self):
         if self.embed and (self.format == u'jpeg' or self.format == u'jpg'):
             return self._data_and_metadata()
 
+    def _repr_bmp_(self):
```

Comment:
I would prefer to avoid adding any new `_repr_<single-type>_` methods, instead opting for any new representations being added via a single arbitrary mime-type method (#10090).